src/components/HeaderPageLayout.tsx

```diff
@@ -1,114 +1,108 @@
         <ScreenWrapper
-            style={[StyleUtils.getBackgroundColorStyle(backgroundColor || theme.appBG)]}
+            style={[StyleUtils.getBackgroundColorStyle(backgroundColor ?? theme.appBG)]}
             shouldEnablePickerAvoiding={false}
             includeSafeAreaPaddingBottom={false}
             offlineIndicatorStyle={[appBGColor]}
             testID={HeaderPageLayout.displayName}
         >
+            {/** @ts-expect-error TODO: Remove once ScreenWrapper (https://github.com/Expensify/App/issues/25128) is migrated to TS */}
             {({safeAreaPaddingBottomStyle}) => (
-                <>
-                    <HeaderWithBackButton
-                        // eslint-disable-next-line react/jsx-props-no-spreading
-                        {...propsToPassToHeader}
-                        titleColor={titleColor}
-                        iconFill={iconFill}
-                    />
-                    <View style={[styles.flex1, appBGColor, !isOffline && !_.isNull(footer) ? safeAreaPaddingBottomStyle : {}]}>
+                <HeaderWithBackButton
+                    // eslint-disable-next-line react/jsx-props-no-spreading
+                    {...propsToPassToHeader}
+                    titleColor={titleColor}
+                    iconFill={iconFill}
+                >
+                    <View style={[styles.flex1, appBGColor, !isOffline && !(footer === null) ? safeAreaPaddingBottomStyle : {}]}>
                         {/** Safari on ios/mac has a bug where overscrolling the page scrollview shows green background color. This is a workaround to fix that. https://github.com/Expensify/App/issues/23422 */}
                         {Browser.isSafari() && (
                             <View style={styles.dualColorOverscrollSpacer}>
-                                <View style={[styles.flex1, StyleUtils.getBackgroundColorStyle(backgroundColor || theme.appBG)]} />
+                                <View style={[styles.flex1, StyleUtils.getBackgroundColorStyle(backgroundColor ?? theme.appBG)]} />
                                 <View style={[isSmallScreenWidth ? styles.flex1 : styles.flex3, appBGColor]} />
                             </View>
                         )}
-                        <ScrollView
-                            contentContainerStyle={[safeAreaPaddingBottomStyle, style, scrollViewContainerStyles]}
-                            offlineIndicatorStyle={[appBGColor]}
-                        >
-                            {!Browser.isSafari() && <View style={styles.overscrollSpacer(backgroundColor || theme.appBG, windowHeight)} />}
-                            <View style={[styles.alignItemsCenter, styles.justifyContentEnd, StyleUtils.getBackgroundColorStyle(backgroundColor || theme.appBG), ...headerContainerStyles]}>
+                        <ScrollView contentContainerStyle={[safeAreaPaddingBottomStyle, style, scrollViewContainerStyles, appBGColor]}>
```

Comment:
Why was `appBGColor` added to `contentContainerStyle`?

Reply:
It was previously applied to `offlineIndicatorStyle ` but you are right - it was probably not used.